core/primitives/src/types.rs

```diff
@@ -24,8 +24,10 @@ pub type StorageUsage = u64;
 pub type StorageUsageChange = i64;
 /// Nonce for transactions.
 pub type Nonce = u64;
-/// Index of the block.
+/// Height of the block from genesis.
```

Comment:
Well, "from genesis" is not completely accurate, as we preserve the block height on hard forks while the genesis gets changed.